### ISSUE TITLE
[Crash] group list crashes

### ISSUE THREAD

**Issue by ildar** [zhanghai/MaterialFiles#187]
File Properties Permissions Groups

**Comment by zhanghai** [comment]
What device and which version of Android are you using? Could you provide the crash stacktrace via things like `adb logcat`?

**Comment by ildar** [comment]
Android 6 and 7, totally different. Logcat comes a bit later.

**Comment by ildar** [comment]
```
11-28 21:05:15.015  8935  8935 E AndroidRuntime: FATAL EXCEPTION: main
11-28 21:05:15.015  8935  8935 E AndroidRuntime: Process:
me.zhanghai.android.files, PID: 8935
11-28 21:05:15.015  8935  8935 E AndroidRuntime:
java.lang.IllegalStateException: Can't access the Fragment View's
LifecycleOwner when getView() is null i.e., before onCreateView() or after
onDestroyView()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
androidx.fragment.app.Fragment.N()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
me.zhanghai.android.files.fileproperties.permissions.SetPrincipalDialogFragment.l(:7)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.c.c()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
androidx.fragment.app.Fragment.g()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q.a(:59)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q.l(:1)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q.a()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.a.d()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q.a(:25)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q.c()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q.d()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at a.k.d.q$d.run()
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
android.os.Handler.handleCallback(Handler.java:751)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
android.os.Handler.dispatchMessage(Handler.java:95)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
android.os.Looper.loop(Looper.java:154)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
android.app.ActivityThread.main(ActivityThread.java:6186)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
java.lang.reflect.Method.invoke(Native Method)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
11-28 21:05:15.015  8935  8935 E AndroidRuntime: at
de.robv.android.xposed.XposedBridge.main(XposedBridge.java:107)
```

**Comment by zhanghai** [comment]
Thanks, it's indeed a regression. Fixed in 0d98dc3.
